qemu/qemu.go

```diff
@@ -513,6 +534,37 @@ func appendKnobs(params []string, config Config) []string {
 	return params
 }
 
+func appendNetDevices(params []string, config Config) []string {
```

Comment:
@sameo longer term the networking setup for VM's will look closer to https://github.com/01org/ciao/blob/master/ciao-launcher/qemu.go#L278 rather than the current tap format.

We need that in order to enable both vhost and mq.

The current format used for tap interfaces is the older deprecated one.

I have a PR pending in netlink where I am adding support of mq tap. 

I can change the current codebase to move to the newer format without mq.
That may help abstract the interface better.